### PR TITLE
Modify browser tests to be compatible with RHMI 1.6

### PR DESCRIPTION
**Pull request by jhellar**
# Description

https://issues.redhat.com/browse/AGMS-1183

### CODE REVIEW

**Comment by b1zzu** [comment]
:eyes:

File: browser/utils/bootstrap.js [hunk 1]
```diff
@@ -39,10 +39,15 @@ afterEach(function() {
...
-async function getMdcUrl() {
+async function getSEUrl() {
   // MDC namespace is targeted by default
-  const mdcRoute = await resource(TYPE.ROUTE, ACTION.GET_ALL);
-  return `https://${mdcRoute.items[0].spec.host}`;
+  const seRoute = await resource(
+    TYPE.ROUTE,
+    ACTION.GET_ALL,
+    null,
+    "openshift-webapp"
```
Comment (b1zzu) [inline browser/utils/bootstrap.js]: I'm trining it with a PDS cluster and the namespace is called only `webapp`, probabbly a different between OSD and PDS, shall we do something for it?

**Comment by b1zzu** [comment]
Also, the Mobile Walkthrough is not present in the 1.6.0-rhpds, should I install the master? or is there a better tag for it?

File: browser/utils/bootstrap.js (hunk 1 above)
Comment (jhellar) [reply browser/utils/bootstrap.js]: will look at that

**Comment by jhellar** [comment]
Did you install 1.6.0 directly from RHPDS catalog? If so, then maybe mobile walkthrough was not there yet. Latest 1.6.0 release is ER4.

File: browser/utils/bootstrap.js (hunk 1 above)
Comment (jhellar) [reply browser/utils/bootstrap.js]: should be fixed now also for PDS clusters

**Pull request merged**